examples/cms-buttercms/pages/api/preview.js

```diff
@@ -1,28 +0,0 @@
-import { getPreviewPostBySlug } from '@/lib/api'
-
-export default async function preview(req, res) {
```

Comment:
Any reason this was removed? It'd be great if it used on-demand ISR as well 😄

Reply:
My understanding is that we no longer support the same method for previewing of a draft post, so there's no longer a getPreviewPostBySlug method.

As for the on-demand ISR - that sounds like a very cool feature! Unfortunately, the main next.js developer that we had for creating the starter is not available, and I'm really hesitant to change anything that's going to affect functionality on that level, as I'm not a next.js developer (or even js developer :sweat_smile: myself). 

We have another version of this starter in our docs and our page that we'd also have to update, as we're trying to keep the two versions (i.e., the one in our docs and the one in the next.js examples) from diverging too greatly. Our main goal with the starter was for someone to be able to set it up super-rapidly using the exact same steps across all stacks that we support, so we'd like to prevent having to set up extra environment variables, etc.

If the on-demand static ISR is an absolute requirement for updating our current example, I can forward that information to my team, but I think our preference at this time would be to not implement it if that's a possibility? It also makes the process of testing/hosting a bit more difficult, due to the need to create a production build and start the production server, and we're trying to keep these starters as easy to drop in as possible.

Reply:
No worries - could be a future improvement!